tests/users/run.sh

```diff
@@ -0,0 +1,282 @@
+#!/bin/bash
+
+# Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+# Use of this source code is governed by a license that can be found
+# in the LICENSE file.
+
+## Global
+source $(dirname "$0")/../share/CommonFunctions.sh
+
+## Specific test functions
+
+## Specific test parameters
+numberOfUsersToTest=10
+numberOfUsersToTestMax=1000
+numberOfKeysToTest=1000
+
+## Tests
+_log "INFO" "Tests start"
+_Prepare
+_StartSiodb
+
+### Starting point users
+_RunSqlAndValidateOutput "select NAME from SYS.SYS_USERS" 'ROOT'
+
+## -------------------------------------------------------------------------
+## Users
+## -------------------------------------------------------------------------
+
+#### Create users
+for ((i = 4096; i < $((4096+${numberOfUsersToTest}+1)); ++i)); do
+_RunSql "create user user_test_1_${i}"
+done
+for ((i = 4096; i < $((4096+${numberOfUsersToTest}+1)); ++i)); do
+_RunSql "create user user_test_2_${i}
+         with STATE = ACTIVE"
+done
+for ((i = 4096; i < $((4096+${numberOfUsersToTest}+1)); ++i)); do
+_RunSql "create user user_test_3_${i}
+         with STATE = INACTIVE"
+done
+for ((i = 4096; i < $((4096+${numberOfUsersToTest}+1)); ++i)); do
+_RunSql "create user user_test_4_${i}
+         with REAL_NAME = 'TestUser ${i} 😀'"
```

Comment:
what for is that extra symbol ` 😀` - is it really supposed to be there?
check all occurrences here and below

Reply:
@cxxman yes, it is to include Unicode in tested strings.